commcare_connect/organization/decorators.py

```diff
@@ -1,31 +1,34 @@
 from functools import wraps
 
-from django.http import Http404, HttpResponseRedirect
+from django.core.exceptions import PermissionDenied
+from django.http import HttpResponseRedirect
 from django.urls import reverse
 
 from .models import UserOrganizationMembership
 
+ALL_ORG_ACCESS = "users.all_org_access"
```

Comment:
It might be nice to make a constant with all of these permissions so we can reference them everywhere consistently.